src/inference/pmmh.jl

```diff
@@ -170,12 +170,12 @@ function sample(  model::Model,
     println("  Accept rate         = $accept_rate;")
 
     if resume_from != nothing   # concat samples
-      pushfirst!(samples, resume_from.value2...)
+      pushfirst!(samples, resume_from.info[:samples]...)
     end
     c = Chain(0.0, samples)       # wrap the result by Chain
```

Comment:
Change to `-Inf`?

Reply:
I'm assuming here you mean change

```julia
c = Chain(0.0, samples)
```

to

```julia
c = Chain(-Inf, samples)
```

I pushed a commit to that effect, but I just want to make sure that was correct.

Reply:
Yes, but honestly I'm not sure what the default value should be. `-Inf` would be consistent with the old behaviour, i.e. `log(0.0)`.